packages/app/src/main/config.ts

```diff
@@ -1,5 +1,7 @@
 const config = {
   applicationName: "Coh2 Game Stats",
+  checkCurrentVersionURL: "https://coh2stats.com/electron-app-version.json",
+  checkCurrentVersionLocalDevURL: "http://localhost:3000/electron-app-version.json",
```

Comment:
FYI once we have it deployed you can also test it by changing the version in package.json